d.configuration.md

```diff
@@ -497,7 +509,7 @@ Alternatively:
 
 ```bash
 # let's get a template easily
-kubectl get sa default -o yaml > sa.yaml
+kubectl get sa myuser -o yaml > sa.yaml
```

Comment:
why is that? unless I'm wrong, the "default" service account most probably exists, so the `kubectl get sa default` will give us a result.

Reply:
You are right! My apologies. It my bad that I misinterpreted the solution. Reverted the same.